iOS/FVM/Framework/DamagedParts/Services/DamagedPartsService.swift

```diff
@@ -66,4 +66,8 @@ internal class DamagedPartsService: DamagedPartsServiceProtocol {
     internal func getSerializedParts() -> String {
         return parser.parse(element: SelectionRoot(selectionArray: repository.getAll(), text: initialSelectionRoot?.mainScreenText ?? ""))
     }
+    
+    internal func getInitialSelectionRoot() -> SelectionRoot? {
```

Comment:
I'd make it a computed property instead of method, what say you?

Reply:
I left it without changes